manifest.json

```diff
@@ -47,7 +47,7 @@
   "custom": {
       "gear-builder": {
           "category": "converter",
-          "image": "flywheel/dicom-send:1.0.0"
+          "image": "flywheel/dicom-send:1.0.1"
```

Comment:
```suggestion
          "image": "flywheel/dicom-send:1.1.1"
```

Reply:
should be 1.1.0, will change in next push